roles/workaround_foreman_remote_execution_core/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Update all packages on the system
+  include_role:
+    name: update_os_packages
+
+- name: Remove rubygem-foreman_remote_execution_core
+  package:
+    state: absent
+    name: rubygem-foreman_remote_execution_core
+  when:
+    - ansible_os_family == "RedHat"
```

Comment:
Since the whole task is now guarded by this, I don't think there's a need to repeat it.